topic-operator/src/main/java/io/strimzi/operator/topic/Config.java

```diff
@@ -159,6 +165,19 @@ private Value(String key, Type<? extends T> type, boolean required) {
     /** The endpoint identification algorithm used by clients to validate server host name. The default value is https. Clients including client connections created by the broker for inter-broker communication verify that the broker host name matches the host name in the broker’s certificate. Disable server host name verification by setting to an empty string.**/
     public static final Value<String> TLS_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM = new Value<>(TC_TLS_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM, STRING, "HTTPS");
 
+    /**
+     * The store topic for the Kafka Streams based TopicStore
+     */
+    public static final Value<String> STORE_TOPIC = new Value<>(TC_STORE_TOPIC, STRING, "store-topic");
```

Comment:
Something like __strimzi_topic_store would be a better default.

Reply:
Yeah, since those log topics get autogenerated from the store name ... will fix it.

Reply:
I got confused here. This is the topic for storing "topics" information so, because it's an internal topic in the infrastructure, @tombentley asked to rename it as `__strimzi_topic_store` while I see this name used for the store name which is not a topic, right?

Reply:
This is the name of the entry topic in our Streams topology.
Where we construct a flow for a Kafka message (that is sent from KafkaStreamsTopicStore), so it ends up in KVStore.

Ah, I see I renamed STORE_NAME (which is the name of our KVStore), but forgot to rename this topic name ... doing it now.

Reply:
@tombentley but it's gonna be "__strimzi_store_topic", since it's a topic not a store.
(you suggested "__strimzi_topic_store", which I used for the STORE_NAME)

Reply:
But what it stores is topics. It seems weird to call a topic "...topic", because obviously it's a topic. I'd be happy with `__strimzi_topic_metadata` or something if we want to distinguish from the kv store name.

Reply:
This topic doesn't store anything.
This is just the name of the topic where out Streams topology "starts",
where we send Kafka messages from KafkaStreamsTopicStore instance.

Reply:
Btw, I'm happy with whatever name you decide. :-)